### PR TITLE
mds cache oversized warning counter

### PR DESCRIPTION
**Pull request by julpark-rh**
# Description
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-XVSP19/

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/reef/cephfs/tier-3_cephfs_bugs.yaml [hunk 1]
```diff
@@ -258,3 +258,9 @@ tests:
       polarion-id: CEPH-83593402
       desc: Additional clone create not started if max concurrent clone creating and snapshot_clone_no_wait is True
       abort-on-fail: false
+  - test:
+    name: mds_chache_oversized_warning_counter
```
Comment (sumabai) [inline suites/reef/cephfs/tier-3_cephfs_bugs.yaml:262]: correct spell error for cache

File: suites/reef/cephfs/tier-3_cephfs_bugs.yaml [hunk 2]
```diff
@@ -258,3 +258,9 @@ tests:
...
+    module: cephfs_bugs.test_mds_cache_oversized_warning_counter.py
+    polarion-id: CEPH-83594160
+    desc: mds_chache_oversized_warning_counter
```
Comment (sumabai) [inline suites/reef/cephfs/tier-3_cephfs_bugs.yaml:265]: same as above

File: tests/cephfs/cephfs_bugs/test_mds_cache_oversized_warning_counter.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                sudo=True, cmd=f"ceph tell mds.{mds_list[0]} cache status"
+            )
+            ceph_status, _ = ceph_status = client1.exec_command(
+                sudo=True, cmd="ceph -s -f json-pretty"
+            )
+            ceph_status = json.loads(ceph_status[0])
+            count = ceph_status["health"]["checks"]["MDS_CACHE_OVERSIZED"]["summary"][
+                "count"
+            ]
+            print(f"Number of count {count}")
```
Comment (sumabai) [inline tests/cephfs/cephfs_bugs/test_mds_cache_oversized_warning_counter.py:92]: please change to log.info

**Comment by hkadam134** [comment]
please include the test in Quincy and Squid also.

File: suites/reef/cephfs/tier-3_cephfs_bugs.yaml (hunk 1 above)
Comment (julpark-rh) [reply suites/reef/cephfs/tier-3_cephfs_bugs.yaml]: corrected

File: tests/cephfs/cephfs_bugs/test_mds_cache_oversized_warning_counter.py (hunk 3 above)
Comment (julpark-rh) [reply tests/cephfs/cephfs_bugs/test_mds_cache_oversized_warning_counter.py]: corrected

**Review by sumabai — APPROVED** [review]

**Review by AmarnatReddy — APPROVED** [review]
LGTM

**Pull request merged**